apps/router-e2e/launch-tests.yml

```diff
@@ -0,0 +1,27 @@
+appId: host.exp.Exponent
+---
+- stopApp: host.exp.Exponent
+- openLink:
+    link: exp://localhost:8081/--/?_TEST_URL=1
```

Comment:
Feed this a URL from the CLI to ensure it's always the expected URL from the dev server.